scripts/ci/stats/merged_prs.py

```diff
@@ -0,0 +1,137 @@
+
+# Script that operates on a merged PR and sends data to elasticsearch for
+# further insepctions using the PR dashboard at
+# https://kibana.zephyrproject.io/
+
+import sys
+import os
+from github import Github
+import argparse
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+from datetime import timedelta
+
+
+date_format = '%Y-%m-%d %H:%M:%S'
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter, allow_abbrev=False)
+
+    parser.add_argument('--pull-request', required=True, help='pull request number', type=int)
+    parser.add_argument('--repo', required=True, help='github repo')
+
+    return parser.parse_args()
+
+def gendata(data, index):
+    for t in data:
+        yield {
+                "_index": index,
+                "_source": t
+                }
+
+def main():
+    args = parse_args()
+    token = os.environ.get('GITHUB_TOKEN')
+    if not token:
+        sys.exit('Github token not set in environment, please set the '
+                 'GITHUB_TOKEN environment variable and retry.')
+
+    gh = Github(token)
+    json_list = []
+    gh_repo = gh.get_repo(args.repo)
+
+    if args.pull_request:
+        pr = gh_repo.get_pull(args.pull_request)
+
+        reviews = pr.get_reviews()
+        print(f'#{pr.number}: {pr.title} - {pr.comments} Comments, reviews: {reviews.totalCount}, {len(pr.assignees)} Assignees (Updated {pr.updated_at})')
+        assignee_reviews = 0
+        reviewers = set()
+        for r in reviews:
+            if r.user and r.state == 'APPROVED':
+                reviewers.add(r.user.login)
+            if pr.assignee and r.user:
+                if r.user.login == pr.assignee.login:
+                    assignee_reviews = assignee_reviews + 1
+        prj = {}
+        # This is all data we get easily though the Github API and serves as the basis
+        # for displaying some trends and metrics.
+        # Data can be extended in the future if we find more information that
+        # is useful through the API
+
+        prj['nr'] = pr.number
+        prj['url'] = pr.url
+        prj['title'] = pr.title
+        prj['comments'] = pr.comments
+        prj['reviews'] = reviews.totalCount
+        prj['assignees'] = len(pr.assignees)
+        prj['updated'] = pr.updated_at.strftime("%Y-%m-%d %H:%M:%S")
+        prj['created'] = pr.created_at.strftime("%Y-%m-%d %H:%M:%S")
+        prj['closed'] = pr.closed_at.strftime("%Y-%m-%d %H:%M:%S")
+        prj['merged_by'] = pr.merged_by.login
+        prj['submitted_by'] = pr.user.login
+        prj['changed_files'] = pr.changed_files
+        prj['additions'] = pr.additions
+        prj['deletions'] = pr.deletions
+        prj['commits'] = pr.commits
+        # The branch we are targeting. main vs release branches.
+        prj['base'] = pr.base.ref
+
+        if pr.assignee:
+            prj['assignee'] = pr.assignee.login
```

Comment:
Does this take into consideration that a PR may have multiple assignees?

Reply:
should now